pkg/virt-controller/services/vm.go

```diff
@@ -192,13 +192,13 @@ func (v *vmService) StartMigration(migration *corev1.Migration, vm *corev1.VM, s
 	if err != nil {
 		return err
 	}
-	_, err = v.KubeCli.CoreV1().Pods(v1.NamespaceDefault).Create(job)
+	_, err = v.KubeCli.CoreV1().Pods(migration.GetObjectMeta().GetNamespace()).Create(job)
 	return err
 }
 
 func (v *vmService) GetMigrationJob(migration *corev1.Migration) (*v1.Pod, bool, error) {
 	selector := migrationJobSelector(migration)
-	podList, err := v.KubeCli.CoreV1().Pods(v1.NamespaceDefault).List(selector)
+	podList, err := v.KubeCli.CoreV1().Pods(migration.ObjectMeta.Namespace).List(selector)
```

Comment:
I'm curious about the mixed usage of the GetObjectMeta() accessor function and using the vm.ObjectMeta directly.

Is there a difference between migration.ObjectMeta.Namespace and migration.GetObjectMeta().getNamespace()?